### ISSUE TITLE
Typescript examples fail to run support scripts with latest CDK releases

### ISSUE THREAD

**Issue by NGL321** [aws-samples/aws-cdk-examples#49]
**Describe the bug**
The support scripts will no longer build with the latest releases of the CDK. They fail with the below error.

**To Reproduce**
Attempt to run either "scripts/build.sh" OR "scripts/build-npm.sh"

**Expected behavior**
 
```
> cdk@0.1.0 build /codebuild/output/src827049164/src/typescript/elasticbeanstalk/elasticbeanstalk-environment 
> tsc 
 
index.ts(13,27): error TS2551: Property 'getContext' does not exist on type 'ConstructNode'. Did you mean 'setContext'? 
npm ERR! code ELIFECYCLE 
npm ERR! errno 2 
npm ERR! cdk@0.1.0 build: `tsc` 
npm ERR! Exit status 2 
npm ERR!  
npm ERR! Failed at the cdk@0.1.0 build script. 
npm ERR! This is probably not a problem with npm. There is likely additional logging output above. 
 
npm ERR! A complete log of this run can be found in: 
npm ERR!     /root/.npm/_logs/2019-06-11T12_38_31_507Z-debug.log 
 
[Container] 2019/06/11 12:38:31 Command did not exit successfully chmod +x scripts/*.sh && scripts/build.sh exit status 1 
[Container] 2019/06/11 12:38:31 Phase complete: BUILD State: FAILED 
[Container] 2019/06/11 12:38:31 Phase context status code: COMMAND_EXECUTION_ERROR Message: Error while executing command: chmod +x scripts/*.sh && scripts/build.sh. Reason: exit status 1 
[Container] 2019/06/11 12:38:31 Entering phase POST_BUILD 
[Container] 2019/06/11 12:38:31 Phase complete: POST_BUILD State: SUCCEEDED 
[Container] 2019/06/11 12:38:31 Phase context status code:  Message:  
```

**Version:**
 - OS: OSX-10.13.6 (High Sierra)
 - Programming Language: Typescript
 - CDK Version: 0.34.0

**Comment by NGL321** [comment]
Fixed with PR #50.
Problem may exist for other language bindings. Will investigate and create new issues appropriately

**Comment by hopewise** [comment]
I have the same issue as here:


```
aws-rails-provisioner build
.
.

running npm run build ...
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! cdk-sample@0.1.0 build: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the cdk-sample@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/samir/.npm/_logs/2020-02-05T11_24_57_799Z-debug.log
```
the log file:
```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/local/Cellar/node/12.12.0/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build'
1 verbose cli ]
2 info using npm@6.11.3
3 info using node@v12.12.0
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle cdk-sample@0.1.0~prebuild: cdk-sample@0.1.0
6 info lifecycle cdk-sample@0.1.0~build: cdk-sample@0.1.0
7 verbose lifecycle cdk-sample@0.1.0~build: unsafe-perm in lifecycle true
8 verbose lifecycle cdk-sample@0.1.0~build: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/samir/Documents/projects/cdk-sample/node_modules/.bin:/Users/samir/.rvm/gems/ruby-2.7.0/bin:/Users/samir/.rvm/gems/ruby-2.7.0@global/bin:/Users/samir/.rvm/rubies/ruby-2.7.0/bin:/Users/samir/.rvm/bin:/usr/local/opt/dart@1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/share/dotnet:~/.dotnet/tools:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/samir/Library/Android/sdk/platform-tools
9 verbose lifecycle cdk-sample@0.1.0~build: CWD: /Users/samir/Documents/projects/cdk-sample
10 silly lifecycle cdk-sample@0.1.0~build: Args: [ '-c', 'tsc' ]
11 silly lifecycle cdk-sample@0.1.0~build: Returned: code: 2  signal: null
12 info lifecycle cdk-sample@0.1.0~build: Failed to exec build script
13 verbose stack Error: cdk-sample@0.1.0 build: `tsc`
13 verbose stack Exit status 2
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:210:5)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:210:5)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
14 verbose pkgid cdk-sample@0.1.0
15 verbose cwd /Users/samir/Documents/projects/cdk-sample
16 verbose Darwin 19.2.0
17 verbose argv "/usr/local/Cellar/node/12.12.0/bin/node" "/usr/local/bin/npm" "run" "build"
18 verbose node v12.12.0
19 verbose npm  v6.11.3
20 error code ELIFECYCLE
21 error errno 2
22 error cdk-sample@0.1.0 build: `tsc`
22 error Exit status 2
23 error Failed at the cdk-sample@0.1.0 build script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 2, true ]
```
